src/vis/sidebar/OpacitySlider.tsx

```diff
@@ -1,26 +1,43 @@
+import { Box, Slider, Stack, Text } from '@mantine/core';
+import { debounce } from 'lodash';
+import { useMemo } from 'react';
+
 import * as React from 'react';
+import { useSyncedRef } from '../../hooks';
 
 interface OpacitySliderProps {
   callback: (n: number) => void;
   currentValue: number;
 }
 
 export function OpacitySlider({ callback, currentValue }: OpacitySliderProps) {
+  const syncedCallback = useSyncedRef(callback);
+
+  const debouncedCallback = useMemo(() => {
+    return debounce((n: number) => syncedCallback.current?.(n), 10);
+  }, [syncedCallback]);
+
   return (
-    <div className="ps-2 pt-0 m-0">
-      <label htmlFor="alphaSlider" className="form-label m-0 p-0">
+    <Stack spacing={0}>
+      <Text weight={500} size={14}>
         Opacity
-      </label>
-      <input
-        type="range"
-        onChange={(e) => callback(+e.currentTarget.value)}
-        className="form-range"
-        value={currentValue}
-        min="=0"
-        max="1"
-        step=".1"
-        id="alphaSlider"
-      />
-    </div>
+      </Text>
+      <Box sx={{ width: '200px' }}>
+        <Slider
+          step={0.05}
+          value={currentValue}
```

Comment:
Please round values to two decimal digits after the decimal point (e.g., `0.30` or even better in this case `0.3`).

![grafik](https://user-images.githubusercontent.com/5851088/195288188-5668d3fb-f4c8-4c37-82c9-2362b0529e67.png)

Reply:
done